### PR TITLE
Add custom PHPCS ruleset `WP_CLI_CS`

### PR DESCRIPTION
**Pull request by pattonwebz**
Sets up the repo ready to use the new WPCliCS rules

Part 1 of #31 

Related https://github.com/wp-cli/wp-cli/issues/5179

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	COMMAND LINE ARGUMENTS
+	For help understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	For help using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage
+	#############################################################################
+	-->
+
+	<!-- What to scan. -->
+	<file>.</file>
+
+	<!-- Ignoring select files/folders.
```
Comment (thrijith) [inline phpcs.xml.dist]: @pattonwebz We would to remove this whole section, this has been done in other repos too.
Ref: https://github.com/wp-cli/maintenance-mode-command/pull/3/commits/6187b8aaae88b1de81895499c67e36032a04b939

File: phpcs.xml.dist [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+	<!-- Strip the filepaths down to the relevant bit. -->
+	<arg name="basepath" value="./"/>
+
+	<!-- Check up to 8 files simultaneously. -->
+	<arg name="parallel" value="8"/>
+
+	<!--
+	#############################################################################
+	USE THE WPCliCS RULESET
```
Comment (thrijith) [inline phpcs.xml.dist]:
```suggestion
	USE THE WP_CLI_CS RULESET
```

File: phpcs.xml.dist [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	#############################################################################
+	-->
+
+	<rule ref="WPCliCS"/>-
```
Comment (thrijith) [inline phpcs.xml.dist]:
```suggestion
	<rule ref="WP_CLI_CS"/>-
```

**Comment by schlessera** [comment]
Ah, you merged one PR into the other, @thrijith . When I now merged #34, this one got merged automatically by Github as well.

We'll have to check on Travis whether the tests actually passed for this one.

**Comment by thrijith** [comment]
> We'll have to check on Travis whether the tests actually passed for this one.

This wouldn't have passed it' own as it is only adding the ruleset but not fixing CS, right?

**Comment by schlessera** [comment]
No, but after merging #34, we could have merged master in here and run the tests in the PR first.

But it's no big deal, and the tests seem to be passing fine anyway.

**Pull request merged**